html-css/week1/index.html

```diff
@@ -307,37 +329,40 @@ <h2 class="experience-sub-title project-title">Payment method</h2>
                 class="project-img"
               />
             </div>
-            <h2 class="experience-sub-title project-title">Knowledge resume</h2>
+            <h3 class="experience-sub-title project-title">Knowledge resume</h3>
             <div class="btn-container">
-              <button
+              <a
                 class="btn btn-color-2 project-btn"
-                onclick="window.open('https://github.com/Bogdy87/Knowledge-Resume', '_blank')"
+                href="https://github.com/Bogdy87/Knowledge-Resume"
+                target="_blank"
               >
                 Github
-              </button>
-              <button
+            </a>
+              <a
                 class="btn btn-color-2 project-btn"
-                onclick="location.href='https://github.com/'"
+                href="https://knowledge-resume87.netlify.app/"
+                target="_blank"
               >
                 Live Demo
-              </button>
+          </a>
             </div>
             
               </button>
             </div>
           </div>
         </div>
       </div>
+      <a href="./#contact">
       <img
         src="./assets/arrow.png"
         alt="Arrow icon"
         class="icon arrow"
-        onclick="location.href='./#contact'"
       />
+    </a>
     </section>
     <section id="contact">
       <p class="section__text__p1">Get in Touch</p>
-      <h1 class="title">Contact Me</h1>
+      <h2 class="title">Contact Me</h2>
```

Comment:
Further down here where you have the link to LinkedIn you can add `target="_blank"`.